torchaudio/models/wav2vec2/utils/import_fairseq.py

```diff
@@ -127,9 +130,10 @@ def import_fairseq_model(original: Module) -> Wav2Vec2Model:
 
```

Comment:
Why there's an underscore after `fairseq`_. ?

Reply:
This is a hyperlink syntax of RST format. See `fairseq` is linked to the repo. https://pytorch.org/audio/master/models.html#import-fairseq-model